argilla-sdk/tests/integration/test_manage_users.py

```diff
@@ -35,6 +35,11 @@ def test_create_user(self, client: Argilla):
         assert user.id is not None
         assert client.users(username=user.username).id == user.id
 
+    def test_create_user_without_password(self, client: Argilla):
+        user = User(username="test_user")
```

Comment:
Should we also test on retrieving an existing user via `client.users("existing_user")`

Reply:
I added a test with your suggestion. Please can you take a look to it?